wagtail/admin/widgets/button.py

```diff
@@ -113,8 +113,11 @@ class PageListingButton(ListingButton):
     aria_label_format = None
     url_name = None
 
-    def __init__(self, *args, page=None, next_url=None, attrs={}, **kwargs):
+    def __init__(
+        self, *args, page=None, next_url=None, attrs={}, page_perms=None, **kwargs
+    ):
         self.page = page
+        self.page_perms = page_perms
```

Comment:
The `register_page_header_buttons` hook is going to need a change of signature imminently anyway (we need to vary the dropdown contents between the index and edit views, because the index page design calls for "Add child page" to be pulled out into its own button - so I'll be adding a `view_name` parameter) so this is probably a good opportunity to get other desirable changes in... will keep that in mind for the next PR!